src/constants/carsInfo.js

```diff
@@ -1,29 +1,78 @@
-const cars = [
-    {
-        id: 1,
-        title: '.BMW',
-        description: 'M5 F90 Competition',
-        date:"2020 г.в.",
-        name:"Artyom",
-        age: "23 года "
-    },
-    {
-        id: 2,
-        title:'.BMW',
-        description: 'M850i Cabrio',
-        date: ' 2021 г.в.',
-        name: "Maria",
-        age: "25 лет "
+import IMAGES from "./carImages";
 
-    },
-    {
-        id: 3,
-        title: '.Volkswagen',
-        description: 'Passat B3 ',
-        date: '1989 г.в.',
-        name: "Joe Biden",
-        age: "78 лет "
-    }
+const cars = [
+  {
+    id: 1,
+    title: "BMW",
+    description: "M5 F90 Competition",
+    transmission: "transmission: automatic",
+    fuel: "fuel: gas",
+    date: "date: 2023",
+    mileage: "mileage: 80023 km.",
+    volume: "volume: 4.4",
```

Comment:
```
    mileage: "mileage: 80023 km.",
    volume: "volume: 4.4",
```

It is very strange... I don't see any reason of this 'syntax'.

Usually it can be 
```

const someObject = {
mark: "iPhone",
model: "13 Pro",
year: 2022,
capacity: 128,
size: {
width: "71.5",
height: "146",
units: "mm"
}
}
```